src/scb/wowi.clj

```diff
@@ -568,13 +588,67 @@
                        (rename-keys rename-map)
                        (update :tag-list (comp vec sort))
                        (update :game-track-list (comp vec sort)))]
+
     addon-data))
 
-(defmethod core/to-catalogue-addon :wowinterface
+(defmethod core/to-addon-summary :wowinterface
   [addon-data-list]
   (if-not (some #{"api--detail.json"} (mapv :filename addon-data-list))
     (warn (format "failed to find API detail, excluding: %s (%s)" (:source-id (first addon-data-list)) (:source (first addon-data-list))))
-    (when-let [addon-data (-to-catalogue-addon addon-data-list)]
+    (when-let [addon-data (-to-addon-summary addon-data-list)]
       (if (s/valid? :addon/summary addon-data)
         addon-data
         (warn (format "failed to coerce addon data to a valid :addon/summary, excluding: %s (%s)" (:source-id addon-data) (:source addon-data)))))))
+
+;;
+
+(defn -to-addon-detail
+  "generates detailed data from the sum of known data about an addon."
+  [addon-data-list]
+  (let [addon-data (sort-filter-merge-addon-data-list addon-data-list)
+
+        addon-data
+        (select-keys addon-data [:latest-release-set
+                                 :wowi/archived-files ;; todo: 'archived-file-list' ?
```

Comment:
do this

Reply:
no, it was done to stay consistent with wowi. when we coerce it to a detail we rename it previous-release-list, which is good enough